app/build.gradle

```diff
@@ -1,57 +0,0 @@
-apply plugin: 'com.android.application'
-
-android {
-    compileSdkVersion 26
-    buildToolsVersion "26.0.1"
-    defaultConfig {
-        applicationId "com.example.kuba10.mypokemonplaces"
-        minSdkVersion 21
-        targetSdkVersion 26
-        versionCode 1
-        versionName "1.0"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-    }
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-        }
-        debug {
-            testCoverageEnabled true
-
-        }
-    }
-}
-
-
-dependencies {
-    compile fileTree(include: ['*.jar'], dir: 'libs')
-    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
-        exclude group: 'com.android.support', module: 'support-annotations'
-    })
-    compile group: 'com.firebaseui', name: 'firebase-ui-database', version: '2.1.0'
-
-
-    compile 'com.google.code.gson:gson:2.7'
-    compile 'com.android.support:appcompat-v7:26.0.0-alpha1'
-    compile 'com.jakewharton:butterknife:8.8.1'
-    compile 'com.android.support:design:26.0.0-alpha1'
-    compile 'com.google.firebase:firebase-database:11.0.4'
-    compile 'com.android.support:support-v4:26.0.0-alpha1'
-    compile 'com.android.support:cardview-v7:26.0.0-alpha1'
-    compile 'com.squareup.retrofit2:retrofit:2.3.0'
-    compile 'com.squareup.picasso:picasso:2.5.2'
-    compile 'com.jakewharton.retrofit:retrofit2-rxjava2-adapter:1.0.0'
-    compile 'io.reactivex.rxjava2:rxandroid:2.0.1'
-    compile 'io.reactivex.rxjava2:rxjava:2.0.9'
-    compile 'com.squareup.retrofit2:converter-gson:2.1.0'
-
-    compile 'com.google.android.gms:play-services-maps:11.0.4'
-    compile 'com.google.android.gms:play-services-location:11.0.4'
-    compile 'com.google.android.gms:play-services-places:11.0.4'
-    testCompile 'junit:junit:4.12'
-    annotationProcessor 'com.jakewharton:butterknife-compiler:8.8.1'
-    apply plugin: 'jacoco-android'
-    apply plugin: 'com.google.gms.google-services'
```

Comment:
Wrzuć te pluginy na samą górę, pod `apply plugin: 'com.android.application'`. Jestem w ogóle zdziwiony, że w takiej wersji się kompiluje :wink:

Reply:
apply plugin: 'com.google.gms.google-services'
naprawde musi byc na samym koncu pliku w ostatnij linii.
Inaczej rzuca:

Error:Execution failed for task ':app:processDebugGoogleServices'.
> Please fix the version conflict either by updating the version of the google-services plugin (information about the latest version is available at https://bintray.com/android/android-tools/com.google.gms.google-services/) or updating the version of com.google.android.gms to 9.0.0.

Reply:
Jasne, tak jak gadaliśmy na żywo: faktycznie na dół, a tutaj chodziło mi bardziej o to, że nie w tych nawiasach